fuzzer/ossfuzz.sh

```diff
@@ -29,7 +29,7 @@ rm -rf ${build_dir}
 mkdir -p ${build_dir}
 pushd ${build_dir}
 
-cmake $SRC/libultrahdr -DUHDR_BUILD_FUZZERS=1
+cmake $SRC/libultrahdr -DUHDR_BUILD_FUZZERS=1 -DUHDR_MAX_DIMENSION=1280 -DUHDR_ENABLE_GLES=1
```

Comment:
this is limiting it way too much for images.

Reply:
With the 1280x1280 resolutions, the run time for a test configuration is around 16s. I wanted to avoid timeouts for now and get the best possible coverage

Reply:
ok, in a follow up CL, let us remove the decoder from encoder fuzzer and plan a separate one.
this is fine for now.